### ISSUE TITLE
Collapse Feature for TreeData

### ISSUE THREAD

**Issue by fanbin** [npcole/npyscreen#32]
I guess the sometimes people(like me) want to collapse particular node of the tree.

**Comment by npcole** [comment]
You can already do that. Try pushing <

**Comment by fanbin** [comment]
That's great. Sorry I failed to find it in the manual. 
One more thing, can I set the default status of nodes to be 'collapsed'?

**Comment by npcole** [comment]
Yes.  Set the attribute .expanded = False on the node of the tree you want
to be collapsed.
